frontend/src/App.js

```diff
@@ -13,51 +13,78 @@ import ResetPasswordPage from "./pages/ResetPasswordPage";
 import HomePage from "./pages/HomePage";
 import WaitingPage from "./pages/WaitingPage";
 import SessionPage from "./pages/SessionPage";
+import ProtectedRoute from "./components/user/ProtectedRoute";
+import VerifyEmailPage from "./pages/VerifyEmailPage";
 
 const theme = createTheme({
-    palette: {
-        primary: {
-            main: "#212121",
-        },
-        secondary: {
-            main: "#FFFFFF",
-        },
-        error: {
-            main: "#ff0000",
-        },
-    },
+	palette: {
+		primary: {
+			main: "#212121",
+		},
+		secondary: {
+			main: "#FFFFFF",
+		},
+		error: {
+			main: "#ff0000",
+		},
+	},
 });
 
 function App() {
-    return (
-        <ThemeProvider theme={theme}>
-            <div className="App">
-                <Router>
-                    <Routes>
-                        <Route
-                            exact
-                            path="/"
-                            element={<Navigate replace to="/signin" />}
-                        ></Route>
-                        <Route path="/signin" element={<SignInPage />} />
-                        <Route path="/signup" element={<CreateAccountPage />} />
-                        <Route
-                            path="/forgotpassword"
-                            element={<ForgotPasswordPage />}
-                        />
-                        <Route
-                            path="/resetpw/:token"
-                            element={<ResetPasswordPage />}
-                        />
-                        <Route path="/home" element={<HomePage />} />
-                        <Route path="/waiting" element={<WaitingPage />} />
-                        <Route path="/session" element={<SessionPage />}/>
-                        
-                    </Routes>
-                </Router>
-            </div>
-        </ThemeProvider>
-    );
+	return (
+		<ThemeProvider theme={theme}>
+			<div className="App">
+				<Router>
+					<Routes>

```

Comment:
![image](https://user-images.githubusercontent.com/47667775/194493132-f877829a-8ec3-427d-b665-8fac484c5dce.png)
I think can have a default path lead to the signin page instead. Can potentially have it to check if user sign it, it defaults to home instead? If its too much trouble, going back to the signin page is a safe bet.